static/app/views/codecov/tests/onboarding.spec.tsx

```diff
@@ -1,18 +1,81 @@
-import {OrganizationFixture} from 'sentry-fixture/organization';
+import {render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
 
-import {render, screen} from 'sentry-test/reactTestingLibrary';
+import TestsOnboardingPage from 'sentry/views/codecov/tests/onboarding';
 
-import TestAnalyticsOnboardingPage from 'sentry/views/codecov/tests/onboarding';
+describe('TestsOnboardingPage', () => {
+  it('renders with GitHub Actions selected by default', () => {
+    render(<TestsOnboardingPage />, {
+      initialRouterConfig: {
+        location: {
+          pathname: '/codecov/tests/new',
+          query: {opt: ''},
```

Comment:
I think it'd be worth adding a test with query being empty/null; subtle but different is that the radio button will have the default value (gha) but the opt will be '' (or undefined if it's just `/new`), leaving the url and button out of sync. Sort of tied to my other comment of having the url be the state vs a useState hook